mobile/native/src/ln_dlc/mod.rs

```diff
@@ -525,6 +532,136 @@ pub fn get_usable_channel_details() -> Result<Vec<ChannelDetails>> {
+                ) {
+                    recover_dlc().await?;
+                } else {
+                    tracing::warn!(
+                        "The DLC is in a state that can not be recovered. Removing position."
+                    );
+                    // maybe a left over after a force-closure
+                    position::handler::update_position_after_dlc_closure(None)?;
+                }
+            }
+        }
+        (None, Some(dlc_channel)) => {
+            if matches!(dlc_channel.state, SubChannelState::OffChainClosed) {
+                tracing::info!("DLC channel and position are in sync");
+            } else {
+                tracing::warn!("Found unexpected sub channel state {:?}", dlc_channel.state);
+                if matches!(dlc_channel.state, SubChannelState::Signed(_)) {
+                    tracing::warn!("Trying to recover position from order");
+                    match order::handler::order_filled() {
+                        Ok(order) => {
+                            let (accept_collateral, expiry_timestamp) = node
+                                .inner
+                                .get_collateral_and_expiry_from(dlc_channel.channel_id)?;
+
+                            position::handler::update_position_after_dlc_creation(
+                                order,
+                                accept_collateral,
+                                expiry_timestamp,
+                            )?;
+
+                            tracing::info!("Successfully recovered position from order.");
+                        }
+                        Err(e) => {
+                            tracing::error!("Could not recover position from order as no filling order was found! Error: {e:#}");
+                        }
+                    }
+                } else if matches!(
+                    dlc_channel.state,
+                    SubChannelState::Offered(_)
+                        | SubChannelState::Accepted(_)
+                        | SubChannelState::Finalized(_)
+                ) {
+                    recover_dlc().await?;
+                } else {
+                    tracing::warn!("The DLC is in a state that can not be recovered.");
+                }
+            }
+        }
+        _ => {}
+    }
+
+    Ok(())
+}
+
+/// Sends a pending RecoverDlc background task notification to the UI, allowing the UI to show a
+/// dialog with a spinner that the DLC protocol is still in progress.
+/// Also triggers the `periodic_check` to process any actions that might have been created after the
+/// channel reestablishment.
+///
+/// fixme(holzeis): We currently use different events for show the recovery of a dlc and the waiting
+/// for an order execution in the happy case (without an restart in between). Those events and
+/// dialogs should be aligned.
+async fn recover_dlc() -> Result<()> {
+    tracing::warn!("It looks like the app was closed while the protocol was not finished yet.");
```

Comment:
```suggestion
    tracing::warn!("It looks like the app was closed while the protocol was still executing.");
```

nit: the other version was not quite right gramatically, `yet` is mostly used with Present Perfect tense.